src/seeds/member.js

```diff
@@ -1 +1,95 @@
-// seeds here
+import mongoose from 'mongoose';
+
+export default [
+  {
+    _id: new mongoose.Types.ObjectId('64650c82fb9b0234915f4be7'),
+    firstName: 'Juan',
+    lastName: 'Perez',
+    dni: 12345678,
+    birthday: '2000-09-28T03:00:00.000Z',
+    phone: '167677839g',
```

Comment:
The phone number must not contain letters.

Reply:
On mongo is like that, with the letter.
![image](https://github.com/BaSP-m2023/nico-megarocket-server/assets/101227384/3a5aa6dd-5b94-41d6-8a72-de8c0bb40ca1)

Reply:
Solved!